tests/electron/playwright.config.ts

```diff
@@ -54,6 +54,7 @@ config.projects.push({
   name: 'electron-api',
   use: {
     browserName: 'chromium',
+    headless: false,
```

Comment:
Seems unrelated.

Reply:
Before Electron / WebView2 tests were treated as headless, so the `headless` fixture was `true` before and my check for the fixed upstream bug was not working as expected.